### DISCUSSION TITLE
✅ iPhone web browser connect to localhost

### DISCUSSION THREAD

**Discussion by hypernice** [isontheline/pro.webssh.net#701] [General]
I can connect over cellular Android firefox using simpleProxy [SOCKS5 127.0.0.1:2025] to my local network using ConnectBot as an SSH Tunnel host@xxx.xxx.xxx.xxx:2000 SOCKS5 source port 2025.

Is it possible to access my local network in the same manner using an iPhone? I can establish an SSH Tunnel using WebSSH, but I am not sure how to proceed using an iPhone browser since it doesn't look like any support adding a proxy configuration.

**Comment by isontheline** [comment]
Hello @hypernice 👋

I think you're talking about https://github.com/isontheline/pro.webssh.net/issues/607

If it's the same need, could you add a thumb 👍 on the issue? 
It will help me to drive the roadmap.

**Reply by hypernice** [reply]
> Do you want to use other apps on your iPhone / iPad to access the SOCKS server?
> 
> The second option is available by using DPF : *:9999 where 9999 is a chopped listening port

It appears that is what I'm looking for. However, I can't figure out how to get Safari on the iphone to direct traffic to the tunnel using DPF.

**Reply by isontheline** [reply]
#607 isn't implemented at this time
